client/sources/ok_test/scheme.py

```diff
@@ -89,7 +89,7 @@ def _import_scheme(self):
             sys.path.insert(0, 'scheme')
             self.scheme = importlib.import_module(self.MODULE)
         except ImportError as e:
-            raise e
+            raise exceptions.ProtocolException('Could not import scheme')
```

Comment:
I think the shorthand for exceptions should be standardized- it should be 'ex', like in the main file.